mods/ra/maps/monster-tank-madness/map.yaml

```diff
@@ -2114,6 +2115,7 @@ Rules:
 		Tooltip:
 			GenericVisibility: Enemy
 			ShowOwnerRow: false
+		AnnounceOnSeen:
```

Comment:
I think this will do nothing on it's own as https://github.com/OpenRA/OpenRA/wiki/Traits#announceonseen everything defaults to false or null.